### PR TITLE
Upgrade/600/server profile templates

### PR DESCRIPTION
**Pull request by sijeesh**
Extending SDK support for OneView API version 600.

Resources;
  Server Profile Templates

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: hpOneView/resources/servers/server_profile_templates.py [hunk 1]
```diff
@@ -143,14 +145,17 @@ def create(self, resource, timeout=-1):
             timeout:
                 Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
                 in OneView, just stop waiting for its completion.
+            force: If set to true, the operation will ignore warnings for SAN storage.
```
Comment (madhav-bharadwaj) [inline hpOneView/resources/servers/server_profile_templates.py]: Description should probably be in the next line, similar to the one above.

File: hpOneView/resources/servers/server_profile_templates.py [hunk 2]
```diff
@@ -218,3 +224,37 @@ def get_transformation(self, id_or_uri, server_hardware_type_uri, enclosure_grou
...
+                 networks will include only networks that share a scope with the server profile template.
+            scopeUris:
+                An expression to restrict the resources returned according to the scopes to which they are assigned.
+            serverHardwareTypeUri:
+                 If the server hardware type specifies a rack server, the list of available network includes all
+                 networks that are applicable for the specified server hardware type. If the server hardware type
+                 specifies a blade server, the enclosureGroupUri parameter must be specified, and the list of
+                 available networks includes all networks that are applicable for the specified server hardware type
+                 and all empty bays within the enclosure group that can support the specified server hardware type.
+            view: The FunctionType (Ethernet or FibreChannel) to filter the list of networks returned.
```
Comment (madhav-bharadwaj) [inline hpOneView/resources/servers/server_profile_templates.py]: Description should probably be in the next line, similar to the one above.

File: hpOneView/resources/servers/server_profile_templates.py (hunk 1 above)
Comment (madhav-bharadwaj) [inline hpOneView/resources/servers/server_profile_templates.py]: Description should probably be in the next line, similar to the one above.

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ Extends support of the SDK to OneView Rest API version 600 (OneView v4.0).
 - SAS interconnect type
 - SAS logical interconnect group
 - SAS logical interconnect
+- Server profile templates
```
Comment (soodpr) [inline CHANGELOG.md]: use Sentence case for this resource.

**Pull request merged**